### PR TITLE
Adding additional step in checkout customization

### PR DESCRIPTION
**Pull request by VinothKumar361**
## Purpose of this pull request

This pull request (PR) - Adding additional step in checkout customization.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/howdoi/checkout/checkout_shipping_methods.html#shipping

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.2/howdoi/checkout/checkout_shipping_methods.md [hunk 1]
```diff
@@ -135,3 +135,14 @@ It overrides the `shippingMethodListTemplate` and `shippingMethodItemTemplate` p
     </body>
 </page>
 ```
+
+## Step 4: Clean the cache and deploy the static contents {#cache-clean-and-static-content-deploy}
```
Comment (jeff-matthews) [inline src/guides/v2.2/howdoi/checkout/checkout_shipping_methods.md]:
```suggestion
## Step 4: Clean the cache and deploy static content
```

File: src/guides/v2.2/howdoi/checkout/checkout_shipping_methods.md [hunk 2]
```diff
@@ -135,3 +135,14 @@ It overrides the `shippingMethodListTemplate` and `shippingMethodItemTemplate` p
...
+
+Clean the cache:
```
Comment (jeff-matthews) [inline src/guides/v2.2/howdoi/checkout/checkout_shipping_methods.md]:
```suggestion
1. Clean the cache:
```

File: src/guides/v2.2/howdoi/checkout/checkout_shipping_methods.md [hunk 3]
```diff
@@ -135,3 +135,14 @@ It overrides the `shippingMethodListTemplate` and `shippingMethodItemTemplate` p
...
+
+`$ bin/magento cache:clean`
+
+deploy the static contents:
```
Comment (jeff-matthews) [inline src/guides/v2.2/howdoi/checkout/checkout_shipping_methods.md]:
```suggestion
1. Deploy static content:
```

File: src/guides/v2.2/howdoi/checkout/checkout_shipping_methods.md [hunk 4]
```diff
@@ -135,3 +135,14 @@ It overrides the `shippingMethodListTemplate` and `shippingMethodItemTemplate` p
     </body>
 </page>
 ```
+
+## Step 4: Clean the cache and deploy the static contents {#cache-clean-and-static-content-deploy}
+
+Clean the cache:
+
+`$ bin/magento cache:clean`
```
Comment (jeff-matthews) [inline src/guides/v2.2/howdoi/checkout/checkout_shipping_methods.md]: 1. Use code fences (```) to wrap the command. 
1. Add `bash` immediately after the first code fence to specify the language for syntax highlighting. 
1. Remove the `$` symbol. Our stylesheet adds the command prompt when we render HTML.

File: src/guides/v2.2/howdoi/checkout/checkout_shipping_methods.md [hunk 5]
```diff
@@ -135,3 +135,14 @@ It overrides the `shippingMethodListTemplate` and `shippingMethodItemTemplate` p
...
+
+deploy the static contents:
+
+`$ bin/magento setup:static-content:deploy`
```
Comment (jeff-matthews) [inline src/guides/v2.2/howdoi/checkout/checkout_shipping_methods.md]: 1. Use code fences (```) to wrap the command. 
1. Add `bash` immediately after the first code fence to specify the language for syntax highlighting. 
1. Remove the `$` symbol. Our stylesheet adds the command prompt when we render HTML.

**Comment by VinothKumar361** [comment]
@jeff-matthews changes made as per your advise

**Pull request merged**